xml/System.Text.Json.Serialization/JsonStringEnumConverter.xml

```diff
@@ -31,7 +39,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Constructor. Creates the <see cref="T:System.Text.Json.Serialization.JsonStringEnumConverter" /> with the default naming policy and allows integer values.</summary>
```

Comment:
```suggestion
        <summary>Creates the <see cref="T:System.Text.Json.Serialization.JsonStringEnumConverter" /> with the default naming policy and allows integer values.</summary>
```

Reply:
How about:
```suggestion
        <summary>Initializes a <see cref="T:System.Text.Json.Serialization.JsonStringEnumConverter" /> instance with the default naming policy and allows integer values.</summary>
```

Reply:
Most of the constructors summaries I've seen have had this sentence format, @Jozkee.